pegasus/sites.v3/code.org/i18n/views/hoc2022_play_activities.hi-IN.md.partial

```diff
@@ -1,7 +1,7 @@
-{{ featured_tutorial, title: "Hello World: Soccer", body: "सॉकर पिच पर इंटरैक्टिव दृश्यों को कोड करते समय आप टीम के कप्तान होते हैं। GOOOAAALLL!", image: "/images/hoc2022-hello-world-soccer.png", url: "https://studio.code.org/s/hello-world-soccer-2022/reset" }}
+{{featured_tutorial, title: “Hello World: Soccer”, body: “सॉकर पिच पर इंटरैक्टिव दृश्यों को कोड करते समय आप टीम के कप्तान होते हैं। GOOOAAALLL!", image: "/images/hoc2022-hello-world-soccer.png", url: "https://studio.code.org/s/hello-world-soccer-2022/reset" }}
```

Comment:
seems like the sync in and up did some fixes that overlap the down and out. Not sure why

Reply:
yes, it is trying to merge some sync down commits that it doesn't see in `staging`... I'm going to rebase the i18n server so it sees the staging-next merges and rerun and see if that cleans up the log

Reply:
actually... gonna try to just rebuild using `staging` since `staging-next` has been merged. let's see what happens.